src/openqdc/datasets/base.py

```diff
@@ -66,16 +71,43 @@ class BaseDataset(torch.utils.data.Dataset):
     __force_methods__ = []
     energy_target_names = []
     force_target_names = []
+    __isolated_atom_energies__ = []
 
-    energy_unit = "hartree"
+    __energy_unit__ = "hartree"
+    __distance_unit__ = "ang"
+    __forces_unit__ = "hartree/ang"
+    __fn_energy__ = lambda x: x
+    __fn_distance__ = lambda x: x
+    __fn_forces__ = lambda x: x
 
-    def __init__(self) -> None:
+    def __init__(
+        self, energy_unit: Optional[str] = None, distance_unit: Optional[str] = None, cache_dir: Optional[str] = None
+    ) -> None:
```

Comment:
I kept the overwrite but we also have a huggingface like way to change the cache dir